crossfire/__init__.py

```diff
@@ -39,3 +42,15 @@ def occurrences(
         max_parallel_requests=max_parallel_requests,
         format=format,
     )
+
+
+def flatten(data, nested_columns=None):
+    nested_columns = set(nested_columns or NESTED_COLUMNS)
+    if not nested_columns.issubset(NESTED_COLUMNS):
+        raise NestedColumnError(nested_columns)
+    if isinstance(data, list):
+        keys = set(data[0].keys()) & nested_columns
```

Comment:
There is a risk that `data` might be an empty list, causing an index error here.

I think that we need an early return in that case ; )

Reply:
I rewrote te function to work with dict. So, when refactoring I will take care of this. So, I wont mark as a resolved conversation...

Reply:
Should I write a specific Error for this early return?

Reply:
Something like "EmptyListError"?

Reply:
> Should I write a specific Error for this early return?

I don't think so. What would you expect as a response for `flatten` data if data is an empty sequence?

Reply:
a.... "empty sequence", perhaps...?

Reply:
I mean, the function should return the data, as it was passed as input...

Reply:
I have commited the changes I have made. Hope this part is fine. otherwise I can fix it.